Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.h

```diff
@@ -71,6 +71,10 @@ class RemoteLayerTreeNode : public CanMakeWeakPtr<RemoteLayerTreeNode> {
     void setActingScrollContainerID(WebCore::GraphicsLayer::PlatformLayerID value) { m_actingScrollContainerID = value; }
     void setStationaryScrollContainerIDs(Vector<WebCore::GraphicsLayer::PlatformLayerID>&& value) { m_stationaryScrollContainerIDs = WTFMove(value); }
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+    void reconcileAfterScroll();
```

Comment:
This doesn't seem to be called "after scroll" as far as I can see. I'd expect a call from `RemoteScrollingCoordinatorProxy::scrollingTreeNodeDidScroll()` or the like, but don't see one.